runtime/devnet/src/lib.rs

```diff
@@ -1030,3 +960,20 @@ cumulus_pallet_parachain_system::register_validate_block! {
 	Runtime = Runtime,
 	BlockExecutor = cumulus_pallet_aura_ext::BlockExecutor::<Runtime, Executive>,
 }
+
+#[cfg(test)]
+mod tests {
+	use crate::Runtime;
+	use std::any::TypeId;
+
+	#[test]
+	fn test_lookup_config() {
```

Comment:
Very nice! Can you just add a comment to say that this ensures that the account if lookup doesnt not perform any state reads? Just so we know why its here and so that it doesnt get removed in the future without reason.

Reply:
Yes good point.